gapir/client/session.go

```diff
@@ -99,11 +99,25 @@ func (s *session) newRemote(ctx context.Context, d remotessh.Device, abi *device
 		return err
 	}
 
+	forceEnableDiskCache := true
+
+	// Add disk cache
```

Comment:
This comment is confusing as the block only might set `forceEnableDiskCache` to false.

Reply:
Will elaborate.